pkgs/by-name/ga/gallery-dl/package.nix

```
@@ -18,7 +18,7 @@ python3Packages.buildPythonApplication {
     owner = "mikf";
     repo = "gallery-dl";
     tag = "v${version}";
-    hash = "sha256-bfE3QUu+ytxd4YkWxsnQ3hTEDE1OgJN8zJxSah2u08I=";
+    hash = "sha256-kw7QBiwq+Mo8vNWmvWLz8U+IAHHvL4oWJY1IA8VBWQs=";
```

```suggestion
    hash = "sha256-wFQtpHkMPscrGhk5tNGc38rPHbFuoDMuvO50KYR+j9M=";
```
Did the previous hash work before?

---

Hmm, it still works for me?

---

Try using `hash = lib.fakeHash;` and rebuilding

---

Can you also run `git diff --no-index` to see the difference between `/nix/store/nvxmpjqbznz6c4830nhk66ivs457wr3a-gallery-dl-1.28.5` and the new build?

---

> Try using `hash = lib.fakeHash;` and rebuilding

I did but got the same hash that causes issues?

<details>
<ul>

```console
$ nix-build -A gallery-dl --pure

these 2 derivations will be built:
  /nix/store/s9xr3lfgh1f87h5x2wx2nsskas0i03lw-source.drv
  /nix/store/aanvc9xwsqfahscccgyljlyfj9vz1m98-gallery-dl-1.28.5.drv
building '/nix/store/s9xr3lfgh1f87h5x2wx2nsskas0i03lw-source.drv'...

trying https://github.com/mikf/gallery-dl/archive/refs/tags/v1.28.5.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  768k    0  768k    0     0   795k      0 --:--:-- --:--:-- --:--:-- 1182k
unpacking source archive /private/tmp/nix-build-source.drv-0/download.tar.gz
error: hash mismatch in fixed-output derivation '/nix/store/s9xr3lfgh1f87h5x2wx2nsskas0i03lw-source.drv':
         specified: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
            got:    sha256-kw7QBiwq+Mo8vNWmvWLz8U+IAHHvL4oWJY1IA8VBWQs=
error: 1 dependencies of derivation '/nix/store/aanvc9xwsqfahscccgyljlyfj9vz1m98-gallery-dl-1.28.5.drv' failed to build
```

</ul>
</details>

For context, I'm doing this on macOS (Darwin)

Obviously, this is an issue on my side, but I'm not sure what's causing it

I will use the correct hash you and the ofborg bot told me tho